src/components/network/network.js

```diff
@@ -271,4 +234,71 @@ export default class Network {
             this.getThreeWindingsTransformer(id)
         );
     }
+
+    makeEqHandler(cb, errHandler, postUpdateCB) {
```

Comment:
please rename to makeEquipmentHandler, longer names are preferable, and eq is often used to mean "equals"

Reply:
done